inc/zoo/swar/SWAR.h

```diff
@@ -94,6 +99,15 @@ struct SWAR {
     constexpr
     SWAR(Literals_t<NBits, T>, const Arg (&values)[N]) : m_v{loadIntoLanes(values)} {}
 
+    constexpr std::array<T, Lanes> to_array() const noexcept {
+        std::array<T, Lanes> result;
+        for (int i = 0; i < Lanes; ++i) {
+            auto otherEnd = Lanes - i - 1;
+            result[otherEnd] = at(i);
+        }
+        return result;
+    }
+
```

Comment:
This endianness conversion bothers me.
Your code is correct, but there is something about endianness that is creeping up that is bothering me.

Reply:
Your code is correct.  Furthermore, your choice to adopt big endianness for arrays was superior to the alternative, but now we have this flipping of endianness that I don't know what to do about; I'm just voicing the idea, and plan to keep it in the back of my head to see where we go with this.